scripts/ci/tools/free-space.py

```diff
@@ -0,0 +1,43 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""free-space.py for cean environment before to start CI"""
+
+import subprocess
+
+exit_code = subprocess.call('./../libraries/_script_init.sh', shell=True)
+print(exit_code)
+
+class style():
```

Comment:
We should use rich's colors scheme and console. See `breeze.py` where I am using rich

Reply:
Oki, Jarek, I will check it

Reply:
wow, this rich.console import Console is amazing, I like it

Reply:
I applied:  `console = Console(force_terminal=True, color_system="standard", width=180)`
@potiuk you applied the same in breeze.py, I just took it.